api/v1alpha3/machinedeployment_types.go

```diff
@@ -232,6 +232,8 @@ func (md *MachineDeploymentStatus) GetTypedPhase() MachineDeploymentPhase {
 // +kubebuilder:subresource:status
 // +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
 // +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase",description="MachineDeployment status such as ScalingUp/ScalingDown/Running/Failed/Unknown"
+// +kubebuilder:printcolumn:name="Targated",type="integer",JSONPath=".status.replicas",description="Total number of non-terminated machines targeted by this deployment"
+// +kubebuilder:printcolumn:name="Available",type="integer",JSONPath=".status.availableReplicas",description="Total number of available machines (ready for at least minReadySeconds)"
 
```

Comment:
Should we also add a printcolumn for "Ready" as well?

Reply:
Done